x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_edit_extension.tsx

```diff
@@ -163,61 +158,57 @@ const WrappedPolicyDetailsForm = memo<{
 
   return (
     <div data-test-subj="endpointIntegrationPolicyForm">
-      {isTrustedAppsByPolicyEnabled ? (
-        <>
-          <div>
-            <EuiText>
-              <h5>
-                <FormattedMessage
-                  id="xpack.securitySolution.endpoint.policyDetails.artifacts.title"
-                  defaultMessage="Artifacts"
-                />
-              </h5>
-            </EuiText>
-            <EuiSpacer size="s" />
-            <FleetTrustedAppsCard
-              policyId={policyId}
-              cardSize="m"
-              customLink={policyTrustedAppsLink}
-            />
-            <EuiSpacer size="s" />
-            <FleetIntegrationHostIsolationExceptionsCard policyId={policyId} />
-          </div>
-          <EuiSpacer size="l" />
-          <div>
-            <EuiText>
-              <h5>
+      <>
```

Comment:
This big change is mostly re-identation. Use the "hide whitespaces" option to see the real change.